.claude/guides/pr.md

```
@@ -0,0 +1,104 @@
+
+## 기본 원칙
+
+- 한국어로 작성
+- 문장 호흡을 짧게 (모바일 가독성 우선)
+- 한 PR = 한 이슈
+
+---
+
+## PR 제목
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)